core/src/main/scala/kafka/coordinator/PidMetadata.scala

```diff
@@ -19,10 +19,17 @@ package kafka.coordinator
 import kafka.utils.nonthreadsafe
 
 @nonthreadsafe
-private[coordinator] class PidMetadata(val pid: Long) {
+private[coordinator] class PidMetadata(val pid: Long,
```

Comment:
Not too sure. Maybe `kafka.transaction.coordinator`? Perhaps can leave the consumer stuff where it is for now.